ambry-router/src/test/java/com.github.ambry.router/GetBlobInfoOperationTest.java

```diff
@@ -414,6 +390,26 @@ public void testBlobAuthorizationFailureOverrideAll() throws Exception {
     testErrorPrecedence(serverErrorCodes, RouterErrorCode.BlobAuthorizationFailure);
   }
 
+  /**
+   * Tests the case where all servers return the same server level error code
+   * @throws Exception
+   */
+  @Test
+  public void testFailureOnServerErrors() throws Exception {
+    List<ServerErrorCode> serverErrors = new ArrayList<>(Arrays.asList(ServerErrorCode.values()));
+    // set the status to various server level level errors (remove all partition level errors or non errors)
```

Comment:
minor: please remove extra `level` in comment.

Reply:
Done